fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/IndexBuilderImpl.java

```diff
@@ -63,12 +63,18 @@
     @Inject
     private ContainmentIndex containmentIndex;
 
+    @Inject
+    private SearchIndex searchIndex;
+
     @Inject
     private TransactionManager transactionManager;
 
     @Inject
     private OcflRepository ocflRepository;
 
+    public IndexBuilderImpl() {
```

Comment:
I don't think this class needs this constructor, does it?

Reply:
Yes - somehow I unintentionally  introduced that.